client/src/components/DashboardLayout.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
Do you need this import? It seems like you're not using it in this file.

Reply:
oo good catch! will remove